cdap-ui/app/features/admin/templates/overview.html

```diff
@@ -22,7 +22,7 @@
       </h3>
     </div>
     <div class="panel-body">
-      <p>This is the control console for CDAP. Here, you can view and manage your system configuration and define and manage namespaces and entities.
+      <p>This is the control console for CDAP. Here, in addition to viewing and managing your system configuration, you can define and manage namespaces and entities.
```

Comment:
Isn't viewing and defining sort of a subset of managing? Maybe `Here, you can manage your system configuration, namespaces, and entities`